### PR TITLE
Add command for finding students by their Covid-19 Status

### LINKED ISSUE #7: As a User, I can filter students by their Status
As a user, I can filter the students by their status, to see which students are Covid-19 Positive or Close Contacts.

### PR DESCRIPTION
**Pull request by lzf834**
Changes made:
- new class FindStatusCommand.java
- new class FindStatusCommandParser.java
- new class StatusContainsKeywordsPredicate.java

Edits:
- AddressBookParser.java to accommodate new Command

Removed:
POTENTIAL_CHANGES.txt used in v1.1

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java [hunk 1]
```diff
@@ -6,15 +6,7 @@
...
-import seedu.address.logic.commands.AddCommand;
-import seedu.address.logic.commands.ClearCommand;
-import seedu.address.logic.commands.Command;
-import seedu.address.logic.commands.DeleteCommand;
-import seedu.address.logic.commands.EditCommand;
-import seedu.address.logic.commands.ExitCommand;
-import seedu.address.logic.commands.FindCommand;
-import seedu.address.logic.commands.HelpCommand;
-import seedu.address.logic.commands.ListCommand;
+import seedu.address.logic.commands.*;
```
Comment (joshuayapwj98) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java]: Need to fix the import statements.

Can use IntelliJ refactoring tool to import single statements.

**Review by joshuayapwj98 — CHANGES_REQUESTED** [review]
Fast Pull request! Just need to look at the import issue which would cause the Checkstyle to flag it. Which will cause the deployment to fail.

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java (hunk 1 above)
Comment (lzf834) [reply src/main/java/seedu/address/logic/parser/AddressBookParser.java]: Fixed! Thanks for the good catch

**Comment by joshuayapwj98** [comment]
Looks good to me! All check styles passed and are ready for merge.

File: src/main/java/seedu/address/logic/commands/FindStatusCommand.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import seedu.address.model.Model;
+import seedu.address.model.person.StatusContainsKeywordsPredicate;
+
+/**
+ * Finds and lists all persons in address book whose status contains any of the argument keywords.
+ * Keyword matching is case insensitive.
+ */
+public class FindStatusCommand extends Command {
+
+    public static final String COMMAND_WORD = "findstatus";
```
Comment (louisdavinlie) [inline src/main/java/seedu/address/logic/commands/FindStatusCommand.java:15]: Should the COMMAND_WORD be find_by_status to follow what we have in our user guide?

**Review by louisdavinlie — COMMENTED** [review]
Looks good. Just a minor doubt on name of command.

**Pull request merged**